static/src/stylesheets/module/commercial/paidfor/_paidfor-content.scss

```diff
@@ -57,10 +57,12 @@
     }
 }
 
-.paidfor-box {
+.paidfor-box--header,
```

Comment:
Is it not worth having this as `paidfor-box` and then adding the modifiers?
As far as I remember the BEM style would be to have `class="paidfor-box paidfor-box--header"` in the HTML?